samples/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+cmake_minimum_required(VERSION 3.13)
+
+project(NovelRTTest VERSION 0.0.1)
+include(GNUInstallDirs)
+
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_SOURCE_DIR}/cmake/Modules/")
+set(CMAKE_CXX_STANDARD 17)
```

Comment:
We should centralize and import several of these options, rather than having to redeclare them in each `CMakeLists.txt`

Reply:
What would be the approach for doing this? Would we have another CMake file that contains these settings or?